codegen.ts

```diff
@@ -0,0 +1,17 @@
+import type { CodegenConfig } from '@graphql-codegen/cli'
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: './src/graphql/data/schema.graphql',
+  documents: ['./src', '!**/node_modules/**', '!**/__mocks__/**', '!**/__generated__/**', '!**/thegraph/**'],
+  generates: {
+    'src/graphql/data/__generated__/types-and-hooks.ts': {
```

Comment:
they recommend checking in this generated file

Reply:
Currently we don't check in any of the generated files, what are your thoughts @zzmp ?

Reply:
In latest it is checked in

Reply:
One thing to note is that the checked in schema file would need to be ignored by our linting

Reply:
in favor of checking it in.